### PR TITLE
Add fixable parameter to fix rules

### LINKED ISSUE #11: [Feature] Add an option to make the rules fixable
I would like to have an option to make the rules fixable when using `tslint --fix`.
The fix will be to remove the first letter (`f` or `x`) for `describe` and `it`.
```json
{
  "no-focused-tests": [true, "fix"],
  "no-disabled-tests": [true, "fix"]
}
```

### PR DESCRIPTION
**Pull request by bastienmoulia**
Fix #11 

I had to change the `test` tree so there is a lot of file change, but the change in `src` is not big.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,3 +26,14 @@ Then reference the rules in your `tslint.json` and enable the rules you want:
...
 }
 ```
+
+## Fixable
+
+The rules `no-focused-tests` and `no-disabled-tests` can have a parameter `fixable` to fix them:
+
+```json
+{
+  "no-focused-tests": [true, "fixable"],
```
Comment (kaiza) [inline README.md:36]: What's the reason for adding the extra parameter? Is there ever a time when you would want to configure the rule not to be fixable?

Comment (kaiza) [reply README.md:36]: I've removed 'fixable' and just always including the fix in the rule. Published as version 1.5.0

Comment (bastienmoulia) [reply README.md:36]: I would like to disable the rules during development and test to keep the `fdescribe` by example. But for the production build I want it to be fix.

Comment (kaiza) [reply README.md:36]: I don’t understand what you mean. I haven’t seen any other rules that are fixable that are configurable like that.

Comment (bastienmoulia) [reply README.md:36]: Indeed I don't think the fixable attribute is interesting for the other rules. But for the test, in development, i like to use fdescribe / fit. But not in prod...
I will use your fixable rules in dev with [tslint-defocus](https://github.com/Sergiioo/tslint-defocus) in prod.

Comment (kaiza) [reply README.md:36]: Still a bit confused. I also like to use fdescribe/fit in development but that still works with these rules. Just don't apply the fix while developing. And if you accidentally commit an fit/fdescribe your prod build process will just fail. Or are you using something that automatically fixes fixable rules?

**Pull request merged**